myhdl/_enum.py

```diff
@@ -153,24 +157,43 @@ def __len__(self):
             return len(self._names)
 
         def __repr__(self):
-            return "<Enum: %s>" % ", ".join(names)
+            return 'enum({})'.format(",".join(["'{}'".format(n) for n in  self._names]))
```

Comment:
The previous version had the space after the comma ", " the changed version does not.

Reply:
The \_\_repr__() method should emit a string that if you would eval it at a prompt it will instantiate an object of this class
from the Python docs:
https://docs.python.org/3/reference/datamodel.html#object.__repr__
>object.__repr__(self)
Called by the repr() built-in function to compute the “official” string representation of an object. If at all possible, this should look like a valid Python expression that could be used to recreate an object with the same value (given an appropriate environment). If this is not possible, a string of the form <...some useful description...> should be returned. The return value must be a string object. If a class defines __repr__() but not __str__(), then __repr__() is also used when an “informal” string representation of instances of that class is required.

>This is typically used for debugging, so it is important that the representation is information-rich and unambiguous.

Hence the *enum()*. The missing space is (obviously) an oversight ... this is why other people must look into the code.

Reply:
Fair enough, thanks for the explanation, it is a good change.